### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Kosmognome** [gh-canon/gh-canon.github.io#1]
Hallo canon,

can you clarify the license of this code?

I am interested in forking it to turn it into a kind of general-use console (not only for SO), but I don't see any license information telling me if that is allowed or under which circumstances it is allowed (attribution, share-alike etc.).

**Comment by Kosmognome** [comment]
Nvm, I found it in the other repo, sorry for the noise

**Comment by gh-canon** [comment]
No problem. I probably need to update licensing/attribution. I've incorporated large portions of Douglas Crockford's json2.js library.

**Comment by gh-canon** [comment]
Yeah, The MIT License (MIT). Apparently JSON is public domain; so, Crockford doesn't require attribution or anything.

btw, I commited some changes to [the main repo](https://github.com/gh-canon/stack-snippet-console) that fixed a bug for IE, Edge, Firefox, etc. Apparently they haven't implemented `Array.prototype.includes()`. I also revised how `HTMLElement` gets serialized. Have a look.
